### ISSUE TITLE
Recommendations

### ISSUE THREAD

**Issue by bengalspice** [dougE83/intro-data-capstone-biodiversity#6]
You were not required to look at the vascular and non-vascular plants as part of the analysis, so it would have been nice if you had focused on the species you were asked about. This recommendation should have been in addition to your recommendation pertaining to reptiles, mammals, and birds.

**Comment by bengalspice** [comment]
Good job on the revision. You described the relationship between mammals, birds, and reptiles correctly. Do you think there are other factors that would help in determining the best conservation action?

**Comment by dougE83** [comment]
Yes there is a lot to derive just from the data at hand, we could look at how the species do in each location to see if one is better off in a different park vs the park they may be at.  Also look into how Amphibian's and Fish tie into the picture as well.
